CODINGSTYLE.md

```diff
@@ -0,0 +1,56 @@
+# Coding Style
+
+## Naming
+
+1. Class names use CamelCase, for example: `class CallProfiler;`.
+   - Qt-related classes prepend an additional `R`, like `class RLine;`.
+     * A class is considered Qt-related if its parent class belongs to the Qt library.
+
+2. Variable names and using-declarations use snake_case:
```

Comment:
It is incorrect to say using-declarations.  The use case you mentioned is type aliasing: https://en.cppreference.com/w/cpp/language/type_alias .  Please correct.